### PR TITLE
[13.0][MIG] document_page_approval

### PR DESCRIPTION
**Pull request by flachica**
@NachoAlesLopez @ivantodorovich Need review

### CODE REVIEW

**Comment by flachica** [comment]
The error is related to #245

**Comment by flachica** [comment]
Testing dependencies. Need help for solve it

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -1,99 +1,68 @@
...
+  rev: v4.3.21
+  hooks:
+  - id: isort
+    name: isort except __init__.py
+    exclude: /__init__\.py$
+- repo: https://github.com/pre-commit/mirrors-eslint
+  rev: v6.5.1
+  hooks:
+  - id: eslint
+    verbose: true
```
Comment (ivantodorovich) [inline .pre-commit-config.yaml:68]: This file shouldn't be here :sweat_smile:

**Review by ivantodorovich — COMMENTED** [review]
I think tests are failing because of the `_compute_is_approval_required` method.
It's giving False when it should be True. Maybe because of changes in the ORM re: computed methods?
It could be this : https://github.com/flachica/knowledge/blob/13.0-mig-document_page_approval/document_page_approval/models/document_page.py#L77 ?

Not sure, though

File: .pre-commit-config.yaml (hunk 1 above)
Comment (flachica) [reply .pre-commit-config.yaml:68]: No, the problem is than in #245 I correct a method dependency of this module

But I need to close this PR. I'm still having problems squashing commits. New PR is comming

Comment (flachica) [reply .pre-commit-config.yaml:68]: New PR #246

**Pull request closed without merging**